platform/security/security-migratable/src/main/java/org/codice/ddf/security/migratable/impl/SecurityMigratable.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.security.migratable.impl;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.codice.ddf.migration.AbstractMigratable;
+import org.codice.ddf.migration.Migratable;
+import org.codice.ddf.migration.MigrationException;
+import org.codice.ddf.migration.MigrationMetadata;
+import org.codice.ddf.migration.MigrationWarning;
+import org.codice.ddf.migration.util.MigratableUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SecurityMigratable extends AbstractMigratable implements Migratable {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SecurityMigratable.class);
+
+    private static final String PDP_POLICIES_DIR = "etc/pdp";
+
+    private static final String CRL_PROP_KEY = "org.apache.ws.security.crypto.merlin.x509crl.file";
+
+    private static final String FILE_CONTAINING_CRL_LOCATION =
+            "etc/ws-security/server/encryption.properties";
+
+    private MigratableUtil migratableUtil;
+
+    public SecurityMigratable(String description, boolean isOptional,
+            MigratableUtil migratableUtil) {
+        super(description, isOptional);
+        this.migratableUtil = migratableUtil;
+    }
+
+    @Override
+    public MigrationMetadata export(Path exportPath) throws MigrationException {
+        Collection<MigrationWarning> migrationWarnings = new ArrayList<>();
+        exportCrlFile(exportPath, migrationWarnings);
+        exportPdpDirectory(exportPath, migrationWarnings);
+        return new MigrationMetadata(migrationWarnings);
+    }
+
+    private void exportCrlFile(Path exportDirectory, Collection<MigrationWarning> migrationWarnings)
+            throws MigrationException {
+        LOGGER.debug("Exporting CRL from property [{}] in file [{}]...",
+                CRL_PROP_KEY,
+                FILE_CONTAINING_CRL_LOCATION);
+        migratableUtil.copyFileFromJavaPropertyValue(Paths.get(FILE_CONTAINING_CRL_LOCATION),
+                CRL_PROP_KEY,
+                exportDirectory,
+                migrationWarnings);
+    }
+
+    /**
```

Comment:
Haha, I was planning on adding javadocs but got pulled away to standup mid-sentence it would seem. You're right, they're probably not needed though, i'll remove it.